release-notes/2024-06/README.md

```diff
@@ -0,0 +1,212 @@
+
+## New Features and Behavior changes
+
+- [Create alarms](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/generating-custom-alarms.html) to notify you when an issue occurs in your system.
+- [Delete one or more product specifications](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/deleting-specifications.html) you no longer need.
+- [Select one or more Test Results and send them to a Jupyter Notebook for analysis](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/analyzing-test-results-with-jupyter-notebooks.html).
+- [Select one or more product specifications and send them to a Jupyter Notebook for analysis](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/analyzing-specification-compliance-with-jupyter-notebooks.html).
+- Automatically copy files linked to a test plan template when you [create a test plan](https://www.ni.com/docs/en-US/bundle/systemlink-enterprise/page/creating-a-test-plan.html) from the template.
+- Filter the test plans grid by a test plan.
+- The DataFrame service now enforces a limit on the number of rows a single data table can have. The limit defaults to 1 billion rows and can be changed by setting the `dataframeservice.ingestion.maxRowCount` value when installing the Helm chart.
+    - When a table reaches the row limit, requests to append more rows to the table will return a `409 Conflict` HTTP error. Query performance for tables with such a large number of rows is highly dependent upon the number of columns, the resources available to Dremio, and the number of executors.
+- The parameters field for notebook execution actions for the Work Order service now supports nested json in addition to strings parameters.
+- AWS specific value are provided to integrate SystemLink with AWS ALB ingresses.
+    - [View this configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-06/getting-started/templates/AWS/aws_supplemental_values.yml)
+
+## Helm Chart Breaking Changes
+
+- `systemlink 0.26.64` Top Level Helm Chart
+    - The Alarm Service was added to the top-level chart.
+        - [View this service configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-06/getting-started/templates/systemlink-values.yaml#L346)
+    - Secrets for the service's MongoDB credentials need to be configured.
+        - [View this secret configuration](https://github.com/ni/install-systemlink-enterprise/blob/2024-06/getting-started/templates/systemlink-secrets.yaml#L180)
+    - The default configuration of workflows TTL Strategy was changed such that `secondsAfterFailure` was replaced with `secondsAfterCompletion`. If these values were overwritten, they might need to be adjusted. If the overwritten values match the default values, they can be removed.
```

Comment:
I don't understand what we are describing here. The line number referenced doesn't include the configuration discussed.

Reply:
@prestwick , I am copying @CiprianAnton since I did ask his input on this documentation when I've added it to the release notes and pasting his input at that time:
- _"The configuration for `secondsAfterFailure` [now renamed `secondsAfterCompletion`] was removed from the starter template altogether. The change can be seen in https://ni.visualstudio.com/DevCentral/_git/Skyline/pullrequest/711308?_a=files&path=/ArgoWorkflows/helm/values.yaml "_
- @CiprianAnton , if you have input on how we could make the documentation clearer given that the configuration was removed from the template? Thanks,

/ArgoWorkflows/helm/values.yaml

![image](https://github.com/ni/install-systemlink-enterprise/assets/127735155/6ad2eb41-bb14-4bf3-99bc-393340b7068d)

Reply:
This breaking change has an impact only on default configuration we were providing. The first is `secondsAfterFailure` was replaced with `secondsAfterCompletion`, which was done in the helm chart. The second one is that we removed this configuration from starter template since it matches the default one defined in the helm, so it does not require attention in normal use cases. The second change is just in the starter template.

Maybe we can phrase this like:
The default configuration of workflows TTL Strategy was changed such that `secondsAfterFailure` was replaced with `secondsAfterCompletion`. The configuration was removed from `getting-started/templates/systemlink-values.yaml` since it matches the default value defined in SystemLink helm chart. If the customer helm values file contains secondsAfterFailure and secondsAfterSuccess set to the default values of 600 and 0, the recommendation is to remove the configuration. If the secondsAfterSuccess does not match default value, the action item is to rename the field to `secondsAfterCompletion`.

@prestwick @CIakab-NI does this sound clearer?

Reply:
@prestwick , I did update the text to the new one suggested by @CiprianAnton and kept the link pointing to the `Configuration for Argo Workflows` section of `getting-started/templates/systemlink-values.yaml`